CHANGELOG.md

```diff
@@ -1,19 +1,20 @@
 # Changelog
 
-## Unreleased
+## [[4.2.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.2.0) - 2022-02-09
 
 ### Added
 
 - Configuration of the border-radius utility class is now possible: `$bitstyles-border-radius-values`, `$bitstyles-border-radius-breakpoints`, `$bitstyles-border-radius-directions`
 - New `u-min-height` utility class
 - `u-height` utility class is now configurable using `$bitstyles-height-values` and `$bitstyles-height-breakpoints`
 - New `u-min-width` utility class
+- New atom component `a-skip-link`, for skip-to-content links
 
 ### Breaking
 
 - `u-round` classes have been renamed `u-border-radius`
 
-## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2021-12-20
+## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2022-22-02
```

Comment:
? I find the year-day-month format a bit weird.

```suggestion
## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2022-02-22
```

Reply:
oh looks like above is also y-m-d format.

Reply:
That was reverse US format apparently 🙈 definitely not what we want. Thanks!